### DISCUSSION TITLE
Hello guys I use docker compose and v 0 [Sep 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3750] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### rror: connect ECONNREFUSED 127.0.0.1:3306   
####     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16) {   
####   errno: 'ECONNREFUSED',   
####   code: 'ECONNREFUSED',   
####   syscall: 'connect',   
####   address: '127.0.0.1',   
####   port: 3306,   
####   fatal: true   
#### }   
#### Error: connect ECONNREFUSED 127.0.0.1:3306   
####     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16) {   
####   errno: 'ECONNREFUSED',   
####   code: 'ECONNREFUSED',   
####   syscall: 'connect',   
####   address: '127.0.0.1',   
####   port: 3306,   
####   fatal: true   
#### }   
#### Idk if this is the reason  
#  
> 🙋‍♂️ **alex.pro**  
#### *this is in logs of NocoDB docker container  
#  
> 🙋‍♂️ **alex.pro**  
#### 2022-09-07T08:37:37.743813Z 16 [Note] Access denied for user 'root'@'localhost' (using password: NO)  
#  
> 🙋‍♂️ **alex.pro**  
#### I see this in MySQL  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/3529  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016989600117424149)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
